.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,51 @@
+
+name: Pull Request
+
+on:
+  #By default, a workflow only runs when a pull_request event's activity type is opened, synchronize, or reopened.
+  pull_request:
+  push:
+    branches:
+      - main
+
+jobs:
+  frontend-tests:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+
+      - uses: actions/setup-node@v3
+        name: Setup Node version 16q
```

Comment:
nitpick" I'll remove the version entirely, leaving only "Setup Node version", since the version is configured inside .nvmrc and can go out of sync, causing confusion

Reply:
Absolutely right, it could cause more than a headache in the future.
Pushing new rev with this change. Thanks!